include/cucumber-cpp/internal/step/BreakpointSupport.hpp

```diff
@@ -0,0 +1,29 @@
+#ifndef CUKE_BREAKPOINTSUPPORT_HPP_
+#define CUKE_BREAKPOINTSUPPORT_HPP_
+
+#include "../CukeDll.hpp"
+
+#include <signal.h>
```

Comment:
It built on Windows, so I assume it'll be fine on all other platforms we care about...